### PR TITLE
[16.0][MIG] mail_debrand

### PR DESCRIPTION
**Pull request by SodexisTeam**

### CODE REVIEW

**Review by flotho — CHANGES_REQUESTED** [review]
Hi,

thanks for this port.
Any chance to also add this commit https://github.com/OCA/social/commit/fc902729bc37a136b11b8f914e777e872c0cd80f

**Comment by StefanRijnhart** [comment]
Please check these bugfixes proposed for 15.0: https://github.com/OCA/social/pull/939

**Comment by atchuthan** [comment]
> Hi,
> 
> thanks for this port. Any chance to also add this commit [fc90272](https://github.com/OCA/social/commit/fc902729bc37a136b11b8f914e777e872c0cd80f)

@flotho Is this commit already available in v15?  If no, I'll first include it in v15 first and then add it to v16 migration.

**Comment by atchuthan** [comment]
> Please check these bugfixes proposed for 15.0: #939

@StefanRijnhart waiting for #939 to be merged.  I'll include this fix after it is merged.

**Comment by atchuthan** [comment]
> > Hi,
> > thanks for this port. Any chance to also add this commit [fc90272](https://github.com/OCA/social/commit/fc902729bc37a136b11b8f914e777e872c0cd80f)
> 
> @flotho Is this commit already available in v15? If no, I'll first include it in v15 first and then add it to v16 migration.

@flotho created a new PR https://github.com/OCA/social/pull/1026 for v15

**Comment by StefanRijnhart** [comment]
@atchuthan thanks for reviewing that one! It's up to you of course, but I think it's alright to already pick them into this branch. You can announce this in the PR description. Without them, the module is a bit broken and this PR is effectively blocked.

File: mail_debrand/README.rst
Comment (Leonidas-VII) [inline mail_debrand/README.rst:29]: Hi, I have read this README and then reviewed it again in the code, it seems Odoo doesn't use the phrase "using Odoo" anymore for version 16.
So I think we should remove the phrase "using Odoo".

File: mail_debrand/readme/DESCRIPTION.rst
Comment (Leonidas-VII) [inline mail_debrand/readme/DESCRIPTION.rst]: I think "using Odoo" is obsolete in ver.16

**Review by StefanRijnhart — APPROVED** [review]
Thanks! I wouldn't mind if you picked https://github.com/OCA/social/pull/1026 into this one already, given that it was included in 14.0 with so much support.

**Comment by atchuthan** [comment]
> Thanks! I wouldn't mind if you picked #1026 into this one already, given that it was included in 14.0 with so much support.

We will create a separate PR for 16.0 once the PR(#1026) for 15.0 is merged in OCA

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Comment by StefanRijnhart** [comment]
pre-commit is now fixed in https://github.com/OCA/social/pull/1065

**Pull request merged**
